client/client.go

```diff
@@ -88,6 +89,39 @@ func (c *LogClient) AddLeaf(ctx context.Context, data []byte) error {
 	}
 }
 
+// GetByIndex returns a single leaf at the requested index.
+func (c *LogClient) GetByIndex(ctx context.Context, index int64) (*trillian.LogLeaf, error) {
+	resp, err := c.client.GetLeavesByIndex(ctx, &trillian.GetLeavesByIndexRequest{
+		LogId:     c.LogID,
+		LeafIndex: []int64{index},
+	})
+	if err != nil {
+		return nil, err
+	}
+	if got, want := len(resp.Leaves), 1; got != want {
+		return nil, fmt.Errorf("len(leaves): %v, want %v", got, want)
+	}
+	return resp.Leaves[0], nil
+}
+
+// ListByIndex returns the requested leaves by index.
+func (c *LogClient) ListByIndex(ctx context.Context, start, count int64) ([]*trillian.LogLeaf, error) {
+	indexes := make([]int64, count)
+	for i := range indexes {
+		indexes[i] = start + int64(i)
+	}
+
+	resp, err := c.client.GetLeavesByIndex(ctx,
```

Comment:
`GetLeavesByIndex` returns `GetLeavesByIndexResponse`.  This API would like to return the `leaves` variable from within `GetLeavesByIndexResponse`.